src/blobifier.py

```diff
@@ -1,17 +1,32 @@
+from typing import Any, List
+
 import numpy as np
 from scipy import ndimage
 from scipy.stats import mode
 
 
+# Assumes that the input data will not use negative values.
```

Comment:
Consider analyzing what data TIFs can actually store and what is reasonable to expect so we can safely make these assumptions.

Reply:
Filed issue #70.